spec/game_spec.rb

```diff
@@ -0,0 +1,149 @@
+require './lib/game'
+
+RSpec.describe Game do
+
+  let(:game) {Game.new({
+    game_id:      "15",
+    season:       "25",
+    type:         "postseason",
+    date_time:    "5/16/13",
+    away_team_id: "3",
+    home_team_id: "6",
+    away_goals:   "2",
+    home_goals:   "5",
+    venue:        "Toyota Stadium",
+    venue_link:   "/api/v1/venues/null"
+    })}
+
+    let(:game_1) {Game.new({
+      game_id:      "30",
+      season:       "70",
+      type:         "postseason",
+      date_time:    "5/20/13",
+      away_team_id: "1",
+      home_team_id: "2",
+      away_goals:   "4",
+      home_goals:   "3",
+      venue:        "Toyota Stadium",
+      venue_link:   "/api/v1/venues/null"
+      })}
+
+      let(:game_2) {Game.new({
+        game_id:      "30",
+        season:       "70",
+        type:         "postseason",
+        date_time:    "5/20/13",
+        away_team_id: "1",
+        home_team_id: "2",
+        away_goals:   "7",
+        home_goals:   "7",
+        venue:        "Toyota Stadium",
+        venue_link:   "/api/v1/venues/null"
+        })}
+
+  describe '#initialize' do
+    it 'exits' do
```

Comment:
Just a little typo right here on 'exists'

Reply:
Fixed. Should be good to go. Nice eyes!